react/ui-components/src/hoc/FieldV1.js

```
@@ -450,6 +481,103 @@ const FieldV1 = ({
+          //         />
+          //         }}
+          //         rules={{ required: required, ...populators.validation }}
+          //         defaultValue={formData?.[populators?.name]}
+          //         name={populators?.name}
+          //         control={controllerProps?.control}
+          //     />
+          //     </div>
+          //   );
+
+            case "multiChildForm":
+            const multichildConfig = populators?.childform || [];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)